gulpfile.js

```diff
@@ -61,14 +62,24 @@ function reloadChromeExtensionsTab(cb) {
     });
 }
 
-function safarify() {
+function babelify() {
     return gulp.src('./build/safari.safariextension/zotero/**/*.js')
         .pipe(buffer())
         .pipe(plumber({errorHandler: onError}))
         .pipe(babel({
             presets: ['es2015']
         }))
-        .pipe(gulp.dest('./build/safari.safariextension/zotero/'));
+        .pipe(gulp.dest((data) => data.base));
+}
+
+function minify() {
+	return gulp.src('./build/**/*.js')
```

Comment:
Only the bookmarklet code gets minified, not connector code. (Can't submit minified code to Mozilla.)